roles/openshift_project_request_template/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+- when: openshift_project_request_template_manage
+  block:
+    - name: Create temp file for template
+      command: mktemp /tmp/openshift-ansible-XXXXXX.yaml
+      register: mktemp
+      changed_when: False
+
+    - name: Generate default project template
+      shell: |
+        {{ openshift.common.client_binary | quote }} \
+          --config {{ openshift.common.config_base | quote }}/master/admin.kubeconfig \
+          --output yaml \
+          adm create-bootstrap-project-template \
+          --name {{ openshift_project_request_template_name | quote }} \
+          > {{ mktemp.stdout | quote }}
+
+    - name: Apply template modifications
+      yedit:
+        state: present
+        src: "{{ mktemp.stdout }}"
+        edits: "{{ openshift_project_request_template_edits }}"
+      when: "openshift_project_request_template_edits | default({}) | length > 0"
```

Comment:
`openshift_project_request_template_edits`'s default value is already set in `defaults/main.yml`. So the ` | default({})` part can be removed here.

Reply:
Removed